saleor/plugins/webhook/tasks.py

```diff
@@ -324,6 +333,18 @@ def send_webhook_using_http(
         AppHeaders.SIGNATURE: signature,
         AppHeaders.API_URL: build_absolute_uri(reverse("api"), domain),
     }
+
+    if custom_headers:
+        try:
+            custom_headers_validator(custom_headers)
```

Comment:
I think we can skip validation at this point, since we have validation at DB field level and mutation level.